metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          "Apache 2.0"
 description      "Installs/Configures td-agent"
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "3.1.0"
+version          "3.1.1"
```

Comment:
3.1.0 is current development/snapshot version and it's not yet released. You don't need to update version number.